File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/NhanVien.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyShopQuanAo
{
    public class NhanVien
    {
        public string maNhanVien { get; set; }      // primary key
        public string tenNhanVien { get; set; }
        public DateTime ngaySinh { get; set; }
        public string diaChi { get; set; }
        public string SDT { get; set; }
        public int namBatDauLam { get; set; }

        // foreign key
        public string maLoaiNhanVien { get; set; }

        // singular
        public LoaiNhanVien loaiNhanVien { get; set; }

        // constructor
        public NhanVien() { }
        public NhanVien(NhanVien nhanVien)
        {
            this.maNhanVien = nhanVien.maNhanVien;
            this.tenNhanVien = nhanVien.tenNhanVien;
            this.ngaySinh = nhanVien.ngaySinh;
            this.diaChi = nhanVien.diaChi;
            this.SDT = nhanVien.SDT;
            this.namBatDauLam = nhanVien.namBatDauLam;
            this.maLoaiNhanVien = nhanVien.maLoaiNhanVien;
            this.loaiNhanVien = nhanVien.loaiNhanVien;
        }
        public NhanVien(string maNhanVien, string tenNhanVien, DateTime ngaySinh, string diaChi, string SDT, int namBatDauLam,
            string maLoaiNhanVien, LoaiNhanVien loaiNhanVien)
        {
            this.maNhanVien = maNhanVien;
            this.tenNhanVien = tenNhanVien;
            this.ngaySinh = ngaySinh;
            this.diaChi = diaChi;
            this.SDT = SDT;
            this.namBatDauLam = namBatDauLam;
            this.maLoaiNhanVien = maLoaiNhanVien;
            this.loaiNhanVien = loaiNhanVien;
        }
    }
}

File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/src/mypack/NhanVien.java
package mypack;

public class NhanVien {
	
	public String maNhanVien;      // primary key    
	public String tenNhanVien;
    public String diaChi;
    public String SDT;
    public int namBatDauLam;
    public String maLoaiNhanVien;    // foreign key

    // singular
    public LoaiNhanVien loaiNhanVien;

    // constructor
    public NhanVien() { }
    public NhanVien(NhanVien nhanVien)
    {
        this.maNhanVien = nhanVien.maNhanVien;
        this.tenNhanVien = nhanVien.tenNhanVien;
        this.diaChi = nhanVien.diaChi;
        this.SDT = nhanVien.SDT;
        this.namBatDauLam = nhanVien.namBatDauLam;
        this.maLoaiNhanVien = nhanVien.maLoaiNhanVien;
        this.loaiNhanVien = nhanVien.loaiNhanVien;
    }
    public NhanVien(String maNhanVien, String tenNhanVien,
    		String diaChi, String SDT, int namBatDauLam,
    		String maLoaiNhanVien, LoaiNhanVien loaiNhanVien)
    {
        this.maNhanVien = maNhanVien;
        this.tenNhanVien = tenNhanVien;
        this.diaChi = diaChi;
        this.SDT = SDT;
        this.namBatDauLam = namBatDauLam;
        this.maLoaiNhanVien = maLoaiNhanVien;
        this.loaiNhanVien = loaiNhanVien;
    }
    
    // getter - setter
    public String getMaNhanVien() {
		return maNhanVien;
	}
	public void setMaNhanVien(String maNhanVien) {
		this.maNhanVien = maNhanVien;
	}
	public String getTenNhanVien() {
		return tenNhanVien;
	}
	public void setTenNhanVien(String tenNhanVien) {
		this.tenNhanVien = tenNhanVien;
	}	
	public String getDiaChi() {
		return diaChi;
	}
	public void setDiaChi(String diaChi) {
		this.diaChi = diaChi;
	}
	public String getSDT() {
		return SDT;
	}
	public void setSDT(String sDT) {
		SDT = sDT;
	}
	public int getNamBatDauLam() {
		return namBatDauLam;
	}
	public void setNamBatDauLam(int namBatDauLam) {
		this.namBatDauLam = namBatDauLam;
	}
	public String getMaLoaiNhanVien() {
		return maLoaiNhanVien;
	}
	public void setMaLoaiNhanVien(String maLoaiNhanVien) {
		this.maLoaiNhanVien = maLoaiNhanVien;
	}
	public LoaiNhanVien getLoaiNhanVien() {
		return loaiNhanVien;
	}
	public void setLoaiNhanVien(LoaiNhanVien loaiNhanVien) {
		this.loaiNhanVien = loaiNhanVien;
	}
}
